db/hybrid_logical_vector.go

```diff
@@ -30,6 +31,85 @@ type Version struct {
 	Value uint64 `json:"version"`
 }
 
+// VersionsDeltas will be sorted by version, first entry will be fill version then after that will be calculated deltas
+type VersionsDeltas []Version
+
+func (vde VersionsDeltas) Len() int { return len(vde) }
+
+func (vde VersionsDeltas) Swap(i, j int) {
+	vde[i], vde[j] = vde[j], vde[i]
+}
+
+func (vde VersionsDeltas) Less(i, j int) bool {
+	if vde[i].Value == vde[j].Value {
+		return vde[i].SourceID < vde[j].SourceID
```

Comment:
For the purpose of computing deltas, does the ordering of sources with matching values matter?  i.e. do we actually need SourceID to be acting as a tiebreaker, or can we just stick to `vde[i].Value < vde[j].Value` below and have equality return Less() = false?

Reply:
Also, do we have unit test coverage for matching Values?

Reply:
I think I did this to align with xdcr approach on it. I have removed it because the more I think about it we don't need to align with them on this part. I have returned false in that scenario. 

Added a new test case to `TestVersionDeltaCalculation` for pv map with two entries that have the same value, leading to 0 deltas being computed.